crypto/hash/hashfuncs.go

```diff
@@ -17,11 +17,17 @@
 package hash
 
 import (
-	"crypto/sha256"
 	"encoding/binary"
 	"hash/fnv"
+
+	//"github.com/minio/sha256-simd"
+	// "crypto/sha256" benchmark is at least 10% faster than "github.com/minio/sha256-simd"
+	"crypto/sha256"
 )
 
+const HashBSize = sha256.Size
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported const HashBSize should have comment or be unexported